.github/workflows/build-mongodb-image.yml

```
@@ -0,0 +1,30 @@
+name: Build and publish MongoDB image to ECR
+
+"on":
```

Huh, your right only just realised that normal this would be "true" in YAML. 

V minor: However GitHub Actions doesn't seem to parse it as such, so for consistency could drop the quotes. 
```suggestion
on:
```

---

Ah fair, I think I only did this to keep the linter quiet in my editor. I should just set it up to use actionlint and then it wouldn't complain. Done :)